src/modules/include/moduleManager.hpp

```diff
@@ -53,6 +53,7 @@ class ModuleManager {
     }
 
     std::shared_ptr<ModuleWrapper> GetModule(const std::string & name);
+    void Init();
```

Comment:
Do we really need to have separated Init and Setup functions? The current code doesn't seem to ask for it... maybe I'm missing the big picture but looks like the additional complexity is not needed at the moment. Could something be moved to the constructor?

Reply:
Done